benchmark/python/utils/benchmark_utils.py

```diff
@@ -133,8 +136,14 @@ def run_benchmark(
                         output.append(
                             torch.sparse_csr_tensor(*sp_out, size=dense_out.shape)
                         )
+                        # Check MPACT and torch eager both return sparse csr output.
+                        if output_type:
```

Comment:
is this test for void methods? in any case, since I had to ask, can you document what the "else" would be here

Reply:
This is for the case when user is not running torch sparse eager mode as output_type is only assigned a value other than None when it's run.